client/app/src/accounts/view/addDelegate.html

```diff
@@ -23,10 +23,17 @@ <h2 translate>Add Delegate </h2>
           </md-card-title>
         </md-card> -->
 
-        <md-input-container class="md-block">
+        <md-input-container class="md-block" ng-if="!addDelegateDialog.data.registeredDelegates[0]">
           <label translate>Delegate name</label>
           <input ng-model="addDelegateDialog.data.delegatename" type="text">
         </md-input-container>
+        
+        <md-input-container class="md-block" ng-if="addDelegateDialog.data.registeredDelegates[0]">
+          <label for="delegate-name">Select delegate: </label>
+          <md-select id="delegate-name" ng-model="addDelegateDialog.data.delegatename">
+            <md-option ng-value="delegate.username" ng-repeat="delegate in addDelegateDialog.data.registeredDelegates | orderBy: 'username'">{{delegate.username}}</md-option>
```

Comment:
Actually the "addDelegateDialog.data.registeredDelegates" is updating every time you open the "Add delegate" dialog, no need to restart the app:
https://github.com/ArkEcosystem/ark-desktop/pull/120/commits/3a886aafddae601a723aa37839d49703c34ca89e#diff-2703b0aec8cc10c1ccd99070b8f63b54R781